modules/admin-ui-picard/app/src/thunks/eventDetailsThunks.js

```diff
@@ -0,0 +1,85 @@
+import {
+    loadEventCommentsInProgress,
+    loadEventCommentsSuccess,
+    loadEventCommentsFailure,
+    saveCommentInProgress,
+    saveCommentDone,
+    saveCommentReplyInProgress,
+    saveCommentReplyDone,
+} from '../actions/eventDetailsActions';
+import axios from "axios";
+
+export const fetchComments = (eventId) => async (dispatch) => {
+    try {
+        dispatch(loadEventCommentsInProgress());
+
+        const commentsData = await axios.get(`admin-ng/event/${eventId}/comments`);
+        const comments = await commentsData.data;
+
+        const commentReasonsData = await axios.get(`admin-ng/resources/components.json`);
+        const commentReasons = (await commentReasonsData.data).eventCommentReasons;
+
+        dispatch(loadEventCommentsSuccess(comments, commentReasons));
+    } catch (e) {
+        dispatch(loadEventCommentsFailure());
+        console.log(e);
+    }
+}
+
+export const saveComment = (eventId, commentText, commentReason) => async (dispatch) => {
+    try {
+        dispatch(saveCommentInProgress());
+
+        const commentSaved = await axios.post(`admin-ng/event/${eventId}/comment`,
+            `text=${commentText}&reason=${commentReason}`);
+        await commentSaved.data;
+
+        dispatch(saveCommentDone());
+        return true;
+    } catch (e) {
+        dispatch(saveCommentDone());
+        console.log(e);
+        return false;
+    }
+}
+
+export const deleteComment = (eventId, commentId) => async (dispatch) => {
```

Comment:
you don't need `dispatch` here, because you not working with redux state and actions here.
If you want to fire a notification then you would need `dispatch`